### PR TITLE
added instructions to make anthabuild alias

### PR DESCRIPTION
**Pull request by gogogadgetbioprocess**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,11 +22,31 @@ or run this command from anywhere:
...
+cat<<EOF>>$HOME/.bash_profile
+alias anthabuild='make -C "$(go list -f '{{.Dir}}' github.com/antha-lang/elements)"'
+EOF
+source ~/.bash_profile
+```
+
+#### Linux:
+
+```sh
+cat<<EOF>>$HOME/.bashrc
```
Comment (ddn0) [inline README.md]: .bash_profile instead?

Comment (gogogadgetbioprocess) [inline README.md]: I found I had to use bashrc on the anthavm

Comment (ddn0) [inline README.md]: It is more correct to use .bashrc in all cases. For maximum safety, you should source .bashrc not .bash_profile

**Comment by gogogadgetbioprocess** [comment]
Done

**Pull request merged**